src/components/TableButtonWrapper/TableButtonWrapper.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+interface TableButtonWrapper {
+  children: React.ReactElement<{
+    onClick?: (e: React.MouseEvent<any>) => void;
+    href?: string;
+  }>;
+}
+
+export const TableButtonWrapper = <T extends HTMLElement>({
```

Comment:
Can we get a comment with some technical explanation why it's implemented this way?

Reply:
Done added comments 👌🏻